src/TLuaInterpreter.cpp

```diff
@@ -1968,16 +1968,19 @@ int TLuaInterpreter::createStopWatch(lua_State* L)
         } else if (lua_type(L, s) == LUA_TSTRING) {
             autoStart = false;
             name = QString::fromUtf8(lua_tostring(L, 1));
+        } else if (lua_type(L, s) == LUA_TNIL) {
+            ; // fallthrough for compatibility with old-style stopwatches in case createStopWatch(nil) is passed
+            // note that 'nil' will still count towards the stack's gettop amount
         } else {
-            lua_pushfstring(L, "createStopWatch: bad argument #%d type (name as string or autostart as boolean are optional, got %s!)", luaL_typename(L, s));
+            lua_pushfstring(L, "createStopWatch: bad argument #%d type (name as string or autostart as boolean are optional, got %s!)", s, luaL_typename(L, s));
```

Comment:
Ouch! :deciduous_tree: :spoon: award for me there and below for not providing that `int`...